pyvista/utilities/helpers.py

```diff
@@ -1446,3 +1446,36 @@ def cubemap(path='', prefix='', ext='.jpg'):
         texture.SetInputDataObject(i, flip.GetOutput())
 
     return texture
+
+def _coerce_points_like_arg(points: Union[Sequence, np.ndarray], copy: bool=True):
```

Comment:
Is this the right module for this function?  Most of the functions here are user-facing helpers.  It also seems out of place in misc, which seems less focused (hence the name).

Reply:
My only concern is that we already have `tests/test_common`, which I think will get confusing.  We could start placing new test modules into a structure that mirrors the pyvista directory, i.e. `tests/core/test_common.py` for tests related to `pyvista/core/common.py`.

Reply:
Better yet, let's rename the existing `tests/test_common.py` to `tests/test_dataset.py` which is already noted in a TODO:

https://github.com/pyvista/pyvista/blob/d303dda0f7939d1de8ee3757aabc067c6f21cce0/tests/test_common.py#L1

Reply:
Hmm, I could swear that we used to have a `core/common.py`, I think that's what @akaszynski meant too. But now I only see a stale annotated coverage file. Any chance that `core/common.py` became `common/dataset.py`? In which case the suggestion is to put this function there. _And_ we should rename the test file as you said.

Reply:
Correction: common.py -> dataobject.py https://github.com/pyvista/pyvista/pull/1160

Reply:
Makes sense.  There is an existing `test_dataobject.py`, which I assume houses all the dataobject related tests.

I'm thinking that these kind of functions should be in `utilities` as they could possibly be used in places outside of `pyvista.core`.  So I'm thinking `pyvista.utilities.common` as the final resting place.

Reply:
I wonder if we can put anything else next to `pyvista.utilities.common`. Looks a bit weird to start a new module with a single helper, but this might sort itself out eventually, we could probably leave it as is right now.